app/views/data_quality_reports/version_four/project/support.haml

```diff
@@ -13,11 +13,14 @@
     %thead
       %tr
         - @data[:headers].each do |t|
+          - next if t == 'Project ID'
```

Comment:
Project IDs weren't visible in the report before, I just had to add them to the plucked fields so I could check if it's confidential (by checking against cached list of confidential IDs). It's kind of hacky, but using an `includes` ended up being way more complicate (at least with my rails skill level :)

If this general approach sounds okay, we could also just show the Project IDs in the table, or drop them from the controller instead of the view.

Reply:
This is fine (as would be including them in the page.)  Just curious since it looked a bit odd.